uo_sys_framework/Core/cfgXEH/init/ClientPreInit.sqf

```diff
@@ -144,7 +144,10 @@ LOG("Client Pre Init");
 [QGVAR(PlayerInitEvent), {
     if (GETMVAR(ViewDistance_Enforce,false)) then {
         setViewDistance GETMVAR(ViewDistance,2500);
+    } else {
```

Comment:
I don't understand the point of this statement. PlayerInit happens only for clients, and if `ViewDistance_Enforce` is false it still sets a viewdistance

Reply:
Is allready changed in the next commit

Reply:
Just comment fixed in most recent commit when you push it. Otherwise, good to know.